### PR TITLE
Add lbpool option in as-http

### PR DESCRIPTION
**Pull request by anson627**
without lbpool, I got 1000+ RPS, along with 10%+ timeout and EADDRNOTAVAIL errors
with lbpool (keep_alive = true), I got 1700+ RPS, no errors
so lbpool is awesome, except it doesn't expose readable stream
@mranney @jcorbin @Raynos @ShanniLi

### CODE REVIEW

**Comment by anson627** [comment]
Also tried ForeverAgent, it's not as nice as lbpool.

File: node/as/http.js [hunk 1]
```diff
@@ -154,10 +158,19 @@ TChannelHTTP.prototype.sendResponse = function send(buildResponse, hres, callbac
         var toBufferErr = errors.HTTPResArg2toBufferError(arg2res.err, {
             head: head
         });
-        callback(toBufferErr, null, null);
+        callback(toBufferErr);
         return null;
     }
     var arg2 = arg2res.value;
+    if (body) {
+        callback(null);
```
Comment (ShanniLi) [inline node/as/http.js]: move callback to after the send

File: node/as/http.js [hunk 2]
```diff
@@ -265,7 +303,7 @@ TChannelHTTP.prototype.forwardToHTTP = function forwardToHTTP(tchannel, options,
     function onResponse(inres) {
         if (!sent) {
             sent = true;
-            outres.sendResponse(inres);
+            outres.sendResponse(inres, null);
```
Comment (ShanniLi) [inline node/as/http.js]: null is not necesssary

**Comment by ShanniLi** [comment]
Do you have data for large payload, e.g., ~1MB?

**Comment by anson627** [comment]
not yet, I'm working on the get benchmark support for as-http. I guess the perf sucks since we are not streaming the response.

File: node/as/http.js (hunk 2 above)
Comment (anson627) [inline node/as/http.js]: I thought we prefer to be explicit and avoiding the undefined

**Comment by ShanniLi** [comment]
lgtm

**Pull request merged**